File: Modul2/Tugas 5/PRAK205-2310817210020-Muhammad_Hasbiyan_Rusyadi.c
#include<stdio.h>
#include<math.h> 
int main () {
    float a, b;
    scanf ("%f %f", &a, &b);
    float c = sqrt( b*b - a*a );
    printf("Alas = %.f\n", c);
    printf("Tinggi = %.f\n", a);
    printf("Keliling = %.f\n", a + b + c);
    printf("Luas = %.f cm^2", 0.5 * c * a); }

File: Modul2/Tugas 5/PRAK205-2310817210020-Muhammad_Hasbiyan_Rusyadi.py
import math
a = input("")
b = input("")
c = round(math.sqrt(int(b)*int(b) - int(a)*int(a)))
k = round(int(a)+int(b)+int(c))
L = round(int(a)*int(c)/ 2)
print(f"Alas = {c} cm")
print(f"keliling = {k} cm")
print(f"luas = {L} cm^2")